tailwind.config.ts

```diff
@@ -22,6 +22,7 @@ export default {
      */
     './src/pages/**/*.{js,jsx,ts,tsx}',
     './src/components/**/*.{js,jsx,ts,tsx}',
+    './src/utilities/**/*.{js,jsx,ts,tsx}',
```

Comment:
TW needs to be able to generate classes from datasource which is in utilities folder